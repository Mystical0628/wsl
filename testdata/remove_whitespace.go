package testpkg

import "fmt"

func RemoveWhitespaceNoComments() { // want "block should not start with a whitespace"

	a := 1
	if a < 2 { // want "block should not start with a whitespace"

		a = 2

	} // want "block should not end with a whitespace"

	switch { // want "block should not start with a whitespace"

	case true:
		fmt.Println("true")

	default:
		fmt.Println("false")

	} // want "block should not end with a whitespace"

	_ = a

} // want "block should not end with a whitespace"
