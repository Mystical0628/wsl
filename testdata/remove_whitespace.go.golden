package testpkg

import "fmt"

func RemoveWhitespaceNoComments() {
	a := 1
	if a < 2 {
		a = 2
	} // want "block should not end with a whitespace"

	switch {
	case true:
		fmt.Println("true")

	default:
		fmt.Println("false")
	} // want "block should not end with a whitespace"

	_ = a
} // want "block should not end with a whitespace"
